sencha-workspace/packages/slate-cbl/src/view/skill/OverviewBody.js

```diff
@@ -0,0 +1,92 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.skill.OverviewBody', {
+    extend: 'Ext.container.Container',
+    xtype: 'slate-cbl-skill-overviewbody',
+    requires: [
+        'Slate.cbl.model.Skill',
+
+        'Ext.form.field.ComboBox',
+        'Ext.data.ChainedStore'
+    ],
+
+    items: [
+        {
+            reference: 'skillStatement',
+
+            xtype: 'component',
+            autoEl: 'p',
+            padding: '16 32 0'
+        },
+        {
+            reference: 'demonstrationsTable',
+
+            xtype: 'component',
+            minHeight: 200,
+            tpl: [
+                '<table class="skill-grid">',
+                    '<thead>',
+                        '<tr class="skill-grid-header-row">',
+                            '<th class="skill-grid-header skill-grid-demo-index">&nbsp;</th>',
```

Comment:
By rules do you mean the CSS classes? Instinct says to take advantage of nested CSS selectors but we've found conclusively that it is better for performance, maintainability, and extensibility to use long namespaced class names mostly flatly instead of deep nested selectors.